mythril/analysis/report.py

```diff
@@ -113,16 +136,45 @@ def as_json(self):
 
     def as_swc_standard_format(self):
         """ Format defined for integration and correlation"""
-        result = {
-            "issues": [
+
+        _issues = []
+        sourceList = []
+
+        for key, issue in self.issues.items():
+
+            if issue.bytecode_hash not in sourceList:
+                idx = len(sourceList)
+                sourceList.append(issue.bytecode_hash)
+            else:
+                idx = sourceList.index(issue.bytecode_hash)
+
+            try:
+                title = SWC_TO_TITLE[issue.swc_id]
+            except KeyError:
+                title = "Unspecified Security Issue"
+
+            _issues.append(
                 {
-                    "swc-id": "SWC-{}".format(issue.swc_id),
-                    "bytecodeOffset": issue.address,
-                    "codeHash": issue.bytecode_hash,
+                    "swcID": issue.swc_id,
+                    "swcTitle": title,
+                    "description": {
+                        "head": issue.description_head,
+                        "tail": issue.description_tail,
+                    },
+                    "severity": issue.severity,
+                    "locations": [{"sourceMap": "%d:1:%d" % (issue.address, idx)}],
```

Comment:
if the source format is bytecode then the location should only be the bytecode offset

Reply:
aren't bytecode offset and address same?